streams/src/api/user.rs

```diff
@@ -1033,3 +1090,50 @@ impl<T> PartialEq for User<T> {
 /// this fact is that two users with the same identity but different transport configurations are
 /// considered equal
 impl<T> Eq for User<T> {}
+
+impl<T> User<T> {
+    pub fn serialize1(&self) -> Result<String> {
```

Comment:
If this is only used inside the test does it need to be defined outside the test module?